docs/raw/push-notification-server.md

```diff
@@ -361,19 +365,18 @@ Where `message` is the encrypted payload of the message and `chat_id` is the
 
 If multiple server are available for a given push notification, only one notification
 MUST be sent.
-In such cases `ack_required` MAY bet set and the sender SHOULD be listening
-on the topic derived from the first 4 bytes of `SHA3-256(ack_required)` and with
-a waku AES symmetric encryption key of `ack_required`.
+The sender SHOULD be listening on the topic derived from the first 4 bytes of `SHA3-256(ack_key)` and with
+a waku AES symmetric encryption key of `ack_key`.
 
-If no response is received 
+If no response is received
 in 3 seconds, it MAY be retried against a different server.
```

Comment:
Why is "3 seconds" a good heuristic?

Reply:
Sounds like a good amount of time :)
I was thinking 1 sec for sending, 1 sec for receiving + 1 sec for processing time should give enough room in most cases, but can be changed

Reply:
Good rule of thumb :) but perhaps technically defendable if we parameterise this in the spec so it can be changed depending on network congestion or node spec.

Reply:
how could we phrase this?
`in a reasonable amount of time given the current network conditions`?

Reply:
Ideally have a config parameter `retry_period = 3s` with exponential backoff?

Reply:
We can make it configurable in the implementation, that's fine, what if instead we word it as:

```
If no response is received
a client SHOULD wait at least 3 seconds, after which the request MAY be retried against a different server
```
That way we only cap for a reasonable minimum and the client is free to wait longer if they wish to do so, taking into accounts network condidtions etc?

Reply:
Sounds good, thanks.